shell/platform/android/io/flutter/plugin/editing/SpellCheckPlugin.java

```diff
@@ -0,0 +1,113 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package io.flutter.plugin.editing;
+
+import android.content.Context;
+import android.view.textservice.SentenceSuggestionsInfo;
+import android.view.textservice.SpellCheckerSession;
+import android.view.textservice.SuggestionsInfo;
+import android.view.textservice.TextInfo;
+import android.view.textservice.TextServicesManager;
+import androidx.annotation.NonNull;
+import androidx.annotation.VisibleForTesting;
+import io.flutter.embedding.engine.systemchannels.SpellCheckChannel;
+import java.util.ArrayList;
+import java.util.Locale;
+
+/** Android implementation of the spell check plugin. */
+public class SpellCheckPlugin implements SpellCheckerSession.SpellCheckerSessionListener {
+
+  @NonNull private final Context mContext;
+  @NonNull private final SpellCheckChannel mSpellCheckChannel;
+  @NonNull private final TextServicesManager tsm;
+  private SpellCheckerSession mSpellCheckerSession;
+
+  @VisibleForTesting @NonNull
+  final SpellCheckChannel.SpellCheckMethodHandler mSpellCheckMethodHandler;
```

Comment:
is the prefix `m` used to signal private properties?

Reply:
Yeah, non-public and non-static properties.